src/plasma/plasma_store.c

```diff
@@ -33,6 +35,8 @@
 #include "malloc.h"
 #include "plasma_store.h"
 
+const char *PERSIST_PATH = "/Users/belugajustin/Downloads/misaka/persisted/";
```

Comment:
make this configurable by the user by passing in an option into the plasma store